### ISSUE TITLE
Close button / link for the "Choose type of content" pop up

### ISSUE THREAD

**Issue by jericyuen** [leekelleher/umbraco-nested-content#67]
Was wondering if someone had added in a close link / button to the pop up after we've clicked on the (+) button.

As currently if i've accidently clicked on the (+) button. I need to select something to be rendered and then delete them after that.

**Comment by Myster** [comment]
you can add this to your nestedcontent.html  line 47

```
<span style="float: right; cursor: pointer;" ng-click="closeNodeTypePicker()"><i class="icon-wrong medium"></i></span>
```

I haven't made a PR, since the css is inline & there's no localisation
I also note there's a delayed-mouseleave attribute with the same method closeNodeTypePicker, but this does't seem to have any effect.

**Comment by leekelleher** [comment]
Closing, as I believe this has been fixed in #50 

(Umbraco v7.4 had renamed the "delayedMouseLeave" event - which broke NC's functionality)
